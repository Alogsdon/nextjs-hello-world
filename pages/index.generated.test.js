import renderer from 'react-test-renderer';
import Home from './index';

const renderTree = (tree) => renderer.create(tree);
describe('<Home>', () => {
  it('should render component', () => {
    expect(renderTree(<Home />).toJSON()).toMatchSnapshot();
  });
});
