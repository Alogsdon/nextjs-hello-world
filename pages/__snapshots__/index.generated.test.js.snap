// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home> should render component 1`] = `
<div>
  <h1>
    index
  </h1>
  <h3>
    global styled
  </h3>
  <h2
    className="css-1tvt44g e1kzrjjz0"
  >
    emotion styled
  </h2>
  <span>
    <input
      onClick={[Function]}
      type="submit"
      value="hit rand API"
    />
    0
  </span>
</div>
`;
